src/bilder/images/concourse/deploy.py

```diff
@@ -173,39 +193,42 @@
         caddy_service(caddy_config=caddy_config, do_reload=caddy_config_changed)
 
 # Install Consul and Vault Agent
-hashicorp_products = [
-    Vault(
-        configuration=VaultAgentConfig(
-            vault=VaultConnectionConfig(
-                address=f"https://active.vault.service.consul:{VAULT_HTTP_PORT}",
-                tls_skip_verify=True,
-            ),
-            auto_auth=VaultAutoAuthConfig(
-                method=VaultAutoAuthMethod(
-                    type="aws",
-                    mount_path="auth/aws",
-                    config=VaultAutoAuthAWS(
-                        role=f"concourse-{concourse_config._node_type}"  # noqa: WPS437
-                    ),
+vault = Vault(
+    configuration=VaultAgentConfig(
+        listener=[
+            VaultListener(type="tcp", address="127.0.0.1:8100", tls_disable=True)
```

Comment:
Should this be a magic number?

Reply:
Probably. I'll take care of that in a follow-up PR